exercises/verify.cpp

```diff
@@ -7,12 +7,69 @@ The number 3 is [not] present in the array.
 */
 
 #include <iostream>
+#include <cstdlib>
+#include <ctime>
+#include <limits>
 using namespace std;
 
+template <class Type>
+Type insert_and_control(Type data) // cin control
+{
+  do
+  {
+    cin >> data;
+    if (cin.fail())
+    {
+      cerr << "Insert a number!" << endl;
+      cin.clear();
+      cin.ignore(numeric_limits<streamsize>::max(), '\n');
+    }
+    else
+      break;
+  } while (true);
+  return data;
+}
+
+int *fill_array(int nArr) // fills the array with random numbers between 0 and 15
+{
+  int *array = new int[nArr];
+  for (int i = 0; i < nArr; i++)
+    array[i] = rand() % 16;
```

Comment:
Le parentesi non guastano

Reply:
Non ho capito 😕

Reply:
Parentesi graffe intendo

So che si può scrivere anche senza però in alcuni casi aumenta la leggibilità ed evita eventuali problemi e fraintendimenti anche al livello di esecuzione

Reply:
sistemo subito!